packages/block-editor/src/components/url-input/index.js

```diff
@@ -224,17 +224,23 @@ class URLInput extends Component {
 	}
 
 	render() {
-		const { value = '', autoFocus = true, instanceId, className } = this.props;
+		const { label, value = '', autoFocus = true, instanceId, className } = this.props;
 		const { showSuggestions, suggestions, selectedSuggestion, loading } = this.state;
+		const id = `inspector-url-input-control-${ instanceId }`;
```

Comment:
We are using "inspector" as part of the id. There is no guarantee that this component will be used in the inspector and in fact in Gutenberg core it is used outside the inspector. I think we should remove the inspector reference and use something like `block-editor-url-input-input-fileld-${ instanceId }`.